homeassistant/components/habitica/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
```

Comment:
Note:
```py
    async_add_entities(entities, True)
```
You can remove the True as the integration already has enough data to get a state